.github/workflows/pr-open.yml

```diff
@@ -106,7 +106,8 @@ jobs:
             triggers: ('db/' 'libs/' 'api/' 'admin/')
           - name: db
             file: db/openshift.deploy.yml
-            overwrite: false
+            overwrite: true
+            parameters: -p OC_NAMESPACE=${{ vars.OC_NAMESPACE }}
```

Comment:
Just for my curiosity.. I don't see 'environment' in template, so this `vars.OC_NAMESPACE` will get the value "a4b31c-test"?

Reply:
this will be dev, if we didn't specify the environment, then it will use the default Github repository environment variable, and the value of `OC_NAMESPACE` is "a4b31c-dev" as repository variable. If we specify environment, it will use GitHub environment variable.